### PR TITLE
Prefix all ENVs with VULCAN_ to ensure uniqueness

### PR DESCRIPTION
**Pull request by rbclark**
Standardize naming of others

### CODE REVIEW

File: config/vulcan.default.yml [hunk 1]
```diff
@@ -11,32 +11,32 @@ defaults: &defaults
...
+      domain: <%= ENV['VULCAN_SMTP_DOMAIN'] %>
+      authentication: <%= ENV['VULCAN_SMTP_AUTHENTICATION']&.to_sym %>
+      tls: <%= ActiveModel::Type::Boolean.new.cast(ENV['VULCAN_SMTP_TLS']) %>
+      openssl_verify_mode: <%= ENV['VULCAN_SMTP_OPENSSL_VERIFY_MODE'] %>
+      enable_starttls_auto: <%= ActiveModel::Type::Boolean.new.cast(ENV['VULCAN_SMTP_ENABLE_STARTTLS_AUTO']) %>
+      user_name: <%= ENV['VULCAN_SMTP_SMTP_SERVER_USERNAME'] %>
+      password: <%= ENV['VULCAN_SMTP_SMTP_SERVER_PASSWORD'] %>
   local_login:
-    enabled: <%= ENV['ENABLE_LOCAL_LOGIN'] || true %>
-    email_confirmation: true
```
Comment (jsa5593) [inline config/vulcan.default.yml:27]: Shouldn't the default have email_confirmation enabled

File: config/vulcan.default.yml [hunk 2]
```diff
@@ -11,32 +11,32 @@ defaults: &defaults
...
-      user_name: <%= ENV['MAILER_SMTP_SERVER_USERNAME'] %>
-      password: <%= ENV['MAILER_SMTP_SERVER_PASSWORD'] %>
+      address: <%= ENV['VULCAN_SMTP_ADDRESS'] %>
+      port: <%= ENV['VULCAN_SMTP_PORT'] %>
+      domain: <%= ENV['VULCAN_SMTP_DOMAIN'] %>
+      authentication: <%= ENV['VULCAN_SMTP_AUTHENTICATION']&.to_sym %>
+      tls: <%= ActiveModel::Type::Boolean.new.cast(ENV['VULCAN_SMTP_TLS']) %>
+      openssl_verify_mode: <%= ENV['VULCAN_SMTP_OPENSSL_VERIFY_MODE'] %>
+      enable_starttls_auto: <%= ActiveModel::Type::Boolean.new.cast(ENV['VULCAN_SMTP_ENABLE_STARTTLS_AUTO']) %>
+      user_name: <%= ENV['VULCAN_SMTP_SMTP_SERVER_USERNAME'] %>
```
Comment (jsa5593) [inline config/vulcan.default.yml:23]: SMTP is repetitive in the ENV names.

File: config/vulcan.default.yml [hunk 3]
```diff
@@ -11,32 +11,32 @@ defaults: &defaults
   welcome_text: <%= ENV['VULCAN_WELCOME_TEXT'] || nil %>
   contact_email: <%= ENV['CONTACT_EMAIL']%>
```
Comment (jsa5593) [inline config/vulcan.default.yml:12]: Should be 'VULCAN_CONTACT_EMAIL'

File: config/vulcan.default.yml (hunk 2 above)
Comment (rbclark) [reply config/vulcan.default.yml]: Yep my bad, fixed

File: config/vulcan.default.yml (hunk 3 above)
Comment (rbclark) [reply config/vulcan.default.yml]: 👍

**Pull request merged**